### PR TITLE
Illustrations wrapup

### PR DESCRIPTION
**Pull request by sudara**
Sudara
* [x] Export new 500 page with smoke going out to the right
* [x] Export Island Uke light/dark for home page hero
* [x] Add home page hero html

Ribs
* [x] CSS love on http://localhost:3000/500 — On desktop, smoke should go to the end of the browser window. On mobile, the intention is to crop so we only see the guitar pedal gnome, not scale the image down. See https://github.com/sudara/alonetone/issues/842#issue-504805070
* [x] contributor-less-noise.jpg has the wrong background color on dark theme Fixed by #974
* [x] Home page hero light / dark ala https://www.figma.com/file/YdjrVsNumbBsWVo82Wje2h6N/alonetone-white-theme?node-id=1730%3A820
* [x] Make maintenance page mobile friendly. See http://localhost:3000/maint.html — the css is also inside that file — the image comes out of public/images, as it will be served by nginx, not rails (as rails will be down). The goal is to have image as large as possible. On desktop, it looks best with dark gutters. On mobile, we could make it so the full width of the image matches the width of the device, I guess (could get rid of the dark gutters).

Questions re: css that came up:

1. True or False: It seems like we are fully granular when it comes to colors — meaning, if one adds an element with a color that is different on white/dark, then a line needs to go into dark/white.scss and there is not really any reuse of colors. Some information about this should probably be documented.

2. Colors are grouped by filename dark/white.scss — is the grouping important? It doesn't seem alphabetical. Should we make it that way? The "why" is worth briefly documenting, even if it's just "We are grouping these by filename, as each variable is unique to the file its used in"

3. It looks like there's still a bit of direct color usage by files in components, such as `site_header.scss`. I'm assuming this is the last of em, though?

Basically, looking to have another conversation around moving to css vars, so just want to have my facts straight and have things be clear and clean for any refactoring.

### CODE REVIEW

**Comment by sudara** [comment]
@ofsound There's open work for the 500, the maintenance page and the home page illustration here. I might do the additional blank slates on another PR, just so we don't have too big batches of work!

**Comment by ofsound** [comment]
@sudara 

_1. True or False: It seems like we are fully granular when it comes to colors — meaning, if one adds an element with a color that is different on white/dark, then a line needs to go into dark/white.scss and there is not really any reuse of colors. Some information about this should probably be documented._

True. we are fully granular. I have one solid candidate for a shared color, so that layer may still end up re-appearing.

_2. Colors are grouped by filename dark/white.scss — is the grouping important? It doesn't seem alphabetical. Should we make it that way? The "why" is worth briefly documenting, even if it's just "We are grouping these by filename, as each variable is unique to the file its used in"_

That order was important for me to make the tranisition, certain priority for making sense of that. But at this point the file name sections could just as well be alphabetized.

_3. It looks like there's still a bit of direct color usage by files in components, such as site_header.scss. I'm assuming this is the last of em, though?_

There are a handful of stragglers here and there, perhaps a few more in shadow definitions.
